waterlog/src/Components/Map.js

```diff
@@ -47,75 +49,88 @@ function formatMapData(data) {
   }
 
   markers = data.monitors.map(mon => {
-    return { id: mon.id, lat: mon.lat, lon: mon.long, status: mon.status }
+    return { id: mon.id, lat: mon.lat, lon: mon.long, status: mon.status };
   });
   return { markers, segments };
 }
 
 function generateMapIcons({ segments, markers }, simpleView) {
   const defaultColor = simpleView ? backgroundColor : lighterColor;
   if (!segments || !markers) {
-    return <div></div>
+    return <div />;
   }
-  return segments.map((segment) => {
-    const sensorIn = markers.find(marker => (marker.id === segment.senseIDIn));
-    const sensorOut = markers.find(marker => (marker.id === segment.senseIDOut));
-    if (!sensorIn || !sensorOut) return <div></div>;
+  return segments.map(segment => {
+    const sensorIn = markers.find(marker => marker.id === segment.senseIDIn);
+    const sensorOut = markers.find(marker => marker.id === segment.senseIDOut);
+    if (!sensorIn || !sensorOut) return <div />;
     const validSensor = ["id", "lat", "lon", "status"];
 
-    if (Object.keys(sensorIn) === validSensor || Object.keys(sensorOut) === validSensor) return <div></div>;
+    if (
+      Object.keys(sensorIn) === validSensor ||
+      Object.keys(sensorOut) === validSensor
+    )
+      return <div />;
     let sensorInColor = defaultColor,
       sensorOutColor = defaultColor,
       segmentColor = defaultColor;
-    if (sensorIn.status.toLowerCase() === 'fault') {
+    if (sensorIn.status.toLowerCase() === "fault") {
```

Comment:
Be consistent, either `if (bool) { oneLiner }` or `if () oneLiner`

Reply:
done